src/entity_embedding_utils.jl

```diff
@@ -66,7 +66,7 @@ Returns the indices of the categorical columns in the table `X`.
 function get_cat_inds(X)
     # if input is a matrix; conclude no categorical columns
     Tables.istable(X) || return Int[]
-    types = schema(X).scitypes
+    types = [scitype(Tables.getcolumn(X, name)[1]) for name in Tables.schema(X).names]
     cat_inds = findall(x -> x <: Finite, types)
```

Comment:
I guess you want to replace `X` here with `Tables.coloumns(X)`, because `Tables.getcolumn` and `Tables.schema` are need not be defined for arbitrary tables.

Reply:
Done in 2cc227aba11fdc25bbc4b186b9b96bd2ac763612

Reply:
I think you need it it `Tables.getcolumn(...)` as well. Better, to avoid two conversions, do `Xcol = Tables.columns(X)` and replace all `X` with `Xcol`.

Reply:
Good point. Sorry for missing that.